### PR TITLE
gh-122701: Fix wording of raw strings/bytes in `lexical_analysis.rst`

### PR DESCRIPTION
**Pull request by sobolevn**
* Issue: gh-122701

----
📚 Documentation preview 📚: https://cpython-previews--122702.org.readthedocs.build/

### CODE REVIEW

File: Doc/reference/lexical_analysis.rst [hunk 1]
```diff
@@ -503,7 +503,8 @@ must be expressed with escapes.
    single: r"; raw string literal
 
 Both string and bytes literals may optionally be prefixed with a letter ``'r'``
-or ``'R'``; such strings are called :dfn:`raw strings` and treat backslashes as
+or ``'R'``; such objects are called :dfn:`raw strings`
+and :dfn:`raw bytes` respectively and treat backslashes as
```
Comment (AA-Turner) [inline Doc/reference/lexical_analysis.rst]: Later on (the 3.3 versionadded note) we use the phrase "raw bytes literals", which I think reads better.

```suggestion
and :dfn:`raw bytes literals` respectively and treat backslashes as
```

File: Doc/reference/lexical_analysis.rst [hunk 2]
```diff
@@ -503,7 +503,8 @@ must be expressed with escapes.
...
 literal characters.  As a result, in string literals, ``'\U'`` and ``'\u'``
 escapes in raw strings are not treated specially. Given that Python 2.x's raw
 unicode literals behave differently than Python 3.x's the ``'ur'`` syntax
```
Comment (AA-Turner) [inline Doc/reference/lexical_analysis.rst]: I can't make a suggestion on L511 but that should also go.
```suggestion
literal characters.  As a result, in raw strings, the ``'\U'`` and ``'\u'``
escapes are not treated specially.
```

File: Doc/reference/lexical_analysis.rst (hunk 1 above)
Comment (sobolevn) [reply Doc/reference/lexical_analysis.rst]: @terryjreedy also pointed this out. I think that I will use this term. Thanks!

File: Doc/reference/lexical_analysis.rst [hunk 3]
```diff
@@ -503,11 +503,10 @@ must be expressed with escapes.
...
 
 Both string and bytes literals may optionally be prefixed with a letter ``'r'``
-or ``'R'``; such strings are called :dfn:`raw strings` and treat backslashes as
-literal characters.  As a result, in string literals, ``'\U'`` and ``'\u'``
-escapes in raw strings are not treated specially. Given that Python 2.x's raw
-unicode literals behave differently than Python 3.x's the ``'ur'`` syntax
-is not supported.
+or ``'R'``; such constructs are called :dfn:`raw string literals`
+and :dfn:`raw bytes literals` respectively and treat backslashes as
+literal characters.  As a result, in raw string literals ``'\U'`` and ``'\u'``
```
Comment (AA-Turner) [inline Doc/reference/lexical_analysis.rst]:
```suggestion
literal characters.  As a result, in raw string literals, ``'\U'`` and ``'\u'``
```

**Pull request merged**